src/main/resources/application.yml

```diff
@@ -8,21 +8,20 @@ spring:
             enabled: true
             path: /h2-console
     datasource:
-        url: jdbc:h2:mem:investment;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE
+        url: ${DATASOURSE_URL}
```

Comment:
Define defaults for URL and user. You can reuse original values.
Only the password should not be committed

Reply:
Done